jdaviz/components/tooltip.vue

```diff
@@ -108,6 +108,11 @@ const tooltips = {
   'plugin-dq-show-all': 'Show all quality flags',
   'plugin-dq-hide-all': 'Hide all quality flags',
   'plugin-dq-color-picker': 'Change the color of this DQ flag',
+  'plugin-vo-filter-coverage': `Only show surveys that report coverage within specified radius of provided coordinates. Queries may take longer to process<br /><br />
+    <div style="border: 1px solid gray;" class="pa-2">
+      <strong>Note:</strong>
+      Surveys which have not implemented coverage information will also be excluded. If you are expecting a survey that doesn't appear, try disabling coverage filtering.
+    </div>`
```

Comment:
can this have a max-width or something set?  Right now this results in an extremely wide tooltip which might not fit at smaller resolutions, etc.

Reply:
Done in https://github.com/spacetelescope/jdaviz/pull/2872/commits/213697461fc372b743ed9f2987b8b6e349e7d646